### DISCUSSION TITLE
How to view multiple classes?

### DISCUSSION THREAD

**Discussion by IhsFace** [Rapptz/discord.py#8308] [Q&A]
I have created a command that has buttons and a dropdown menu, but I only want the buttons to appear for one of the options selected in the menu, so I created another class for it. How do I make an interaction message view more than one class?

EDIT:
I am currently using another class with the buttons and the dropdown menu along with the other two classes, I am wondering if there is a way to make it less complicated

**Comment by heysebus** [comment]
Please tell me, how did you use both the buttons and the selection menu in one response to the command. How much I try to do something I just can not understand

**Reply by IhsFace** [reply]
I made a view class that had both the buttons and the dropdown menu, and other view classes that just had either the buttons or the dropdown menu
